feat: learn English block 3.1 (#52548)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b24774d8cdab052ffe2a6.md

---
id: 657b24774d8cdab052ffe2a6
title: Task 57
challengeType: 22
dashedName: task-57
audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
---

<!--
AUDIO REFERENCE:

Alice: I didn't study tech at a traditional university. I'm mostly self-taught, and I took a lot of online courses.
-->

# --description--

> This task serves as a review. You'll practice filling in blanks with words you've already learned. Listen to the dialogue and fill in the blanks correctly.

Reviewer: ```suggestion
Listen to the audio and fill in the blanks correctly.
```
